gateway-release/home/conf/gateway-site.xml

```diff
@@ -139,5 +139,10 @@ limitations under the License.
         <description>The whitelist to be applied for dispatches associated with the service roles specified by gateway.dispatch.whitelist.services.
         If the value is DEFAULT, a domain-based whitelist will be derived from the Knox host.</description>
     </property>
+    <property>
+        <name>gateway.header.x-forward-context.append.servicename</name>
```

Comment:
`private static final String XFORWARDED_ENABLED = GATEWAY_CONFIG_FILE_PREFIX + ".xforwarded.enabled";`

There is an existing config that is `xforwarded`. I think it would be better for the config to be:

`gateway.xforwarded.header.context.append.servicename`

or something like that to make it clear these are related.

Reply:
Good point, that looks much better, thanks !